README.md

```diff
@@ -57,81 +56,114 @@ cd ../server && npm install
+
+
+## Client application / Frontend
+
+On the other hand, the **client application** (frontend) is responsible to do the following:
+- Generate a new local player account
+- Fund the local player account by requesting coins from [Faucet æpp](https://faucet.aepps.com/)
+- POST request to the server application (in order to fetch the mutual channel configuration) by providing:
+  - Account address (ak_...)
+  - Node host (e.g. http://localhost:8000)
+  - Node port (e.g. 3013)
 - Initialize a channel with the given configuration
 - Verify that the deployed contract has the correct source code
 - Respond to other party's actions
-- Execute `channel_close_mutual_tx`
+- Execute `ChannelCloseMutualTx`
 - Reconnect to channel in cases of window reload
 
-# FAQ
-
-## How do I build my own state channel demo?
-
-You can find the steps at the complete [State Channel Demo Tutorial Guide](TUTORIAL.md).
-
-## How do State Channel Demo Game transactions take place?
+Note:
+- After the closing phase, both (client & server application) will send the funds back to the Faucet æpp.
 
-In order to avoid cheating in a turn based implementation (each player move needs to be co-signed in order to be executed), one player hashes his move. The other party receives the off-chain transaction with calldata containing the hash, where he/she can **only** confirm the method opponent called (e.g. opponent picked a move) but not the actual move. 
+# FAQ
 
-After co-signing the opponent's hashed move, the next required step is for the second player to make his/her move. This is achieved by calling the corresponding contract method which does not hash the pick (the opponent's move is already sealed in the channel's state tree).
+## How do I build my own State Channel application?
 
-Now that both players have picked, the contract requires from the first player to reveal his move. Revealing is required as only the first player has the key to the hash. With the execution of it, the winning participant receives the stake. In the case of Draw, the stake is returned to both players.
+You can find the steps at the complete [State Channel Tutorial Guide](TUTORIAL.md).
 
 
-## Is State Channel Demo probably fair?
+## Is State Channel Demo provably fair?
 
 State Channels offer an innovative solution, where off-chain transactions are executed on top of blockchain technology. 
 
-State channels protocol inherently offers security to the demo.
+The State Channel protocol inherently offers security to the demo!
+
+State Channel transactions are ruled by co-signed Smart Contracts. Each party shall verify the content of a transaction before signing it. Those transactions can be off-chain such as `ContractCallTx`, `SpendTx` or on-chain such as `MutualCloseChannelTx`. All transactions need to be co-signed by both parties.
```

Comment:
This is not technically correct. Offchain transactions are performed by applying a list of operations on the offchain state trees, then generating a new `ChannelOffchainTx` with an incremented round and updated state tree root hash. The updates themselves are not included in the `ChannelOffchainTx` for privacy reasons (since after Fortuna). The updates supported are
* `update_transfer` (move amount from one channel participant to the other)
* `update_deposit` (the offchain part of a `ChannelDepositTx`)
* `update_withdrawal` (the offchain part of a `ChannelWithdrawalTx`)
* `update_create_contract`
* `update_call_contract`
Serializations described [here](https://github.com/aeternity/protocol/blob/master/serializations.md#table-of-object-tags) and [here](https://github.com/aeternity/protocol/blob/master/serializations.md#channel-off-chain-update)

In the WebSocket API, these updates correspond to separate method calls: `channels.update.new` (transfer), `channels.update.new_contract`, `channels.update.call_contract`.

There is also a special update op, `data`, which doesn't affect the state trees, but can be used to annotate the update requests. In the WS API, this is called `meta` (see docs [here](https://github.com/aeternity/protocol/blob/master/node/api/channels_api_usage.md#meta-information))

Reply:
updated, but ignored the WS API stuff: https://github.com/aeternity/state-channel-demo/pull/251/commits/05a6d0eb197aadb204779fa2eb970eed1ba3605b